internal/storage/bucket_handle_test.go

```diff
@@ -467,3 +470,372 @@ func (t *BucketHandleTest) TestUpdateObjectMethodWithMissingObject() {
 
 	AssertTrue(errors.As(err, &notfound))
 }
+
+func (t *BucketHandleTest) TestComposeObjectMethodWithDstObjectExist() {
+	// Checking that destination object exist
+	_, err := t.bucketHandle.StatObject(context.Background(),
```

Comment:
Since read assert will work only if the object is present, you can remove the stat call verification.

Reply:
fixed